elastic-container.sh

```diff
@@ -15,92 +15,185 @@
 # More information at https://github.com/peasead/elastic-container"
 
 # Define variables
+ELASTIC_USERNAME="elastic"
 ELASTIC_PASSWORD="password"
 ELASTICSEARCH_URL="http://elasticsearch:9200"
 KIBANA_URL="http://kibana:5601"
-KIBANA_PASSWORD="password"
+LOCAL_KBN_URL="http://127.0.0.1:5601"
```

Comment:
Why are we defining this variable when we already have `KIBANA_URL`?

Reply:
Because curl doesn't work with the dns name